src/Rules/Program.hs

```diff
@@ -19,15 +20,16 @@ import Target
 import UserSettings
 import Util
 
--- | Wrapper is an expression depending on the 'FilePath' to the wrapped binary.
-type Wrapper = FilePath -> Expr String
+-- | Wrapper is an expression depending on the 'FilePath' to the
+-- | library path and name of the wrapped binary.
+type Wrapper = FilePath -> FilePath -> Expr String
```

Comment:
It is name, and will be appended into a path with `libdir -/- "bin"` in the wrapper code.